x-pack/plugins/security_solution/public/overview/components/detection_response/hooks/use_navigate_to_timeline.tsx

```diff
@@ -35,15 +38,28 @@ export const useNavigateToTimeline = () => {
     );
     // Only show detection alerts
     // (This is required so the timeline event count matches the prevalence count)
-    dispatch(
-      sourcererActions.setSelectedDataView({
-        id: SourcererScopeName.timeline,
-        selectedDataViewId: 'security-solution-default',
-        selectedPatterns: ['.alerts-security.alerts-default'],
-      })
-    );
+    // dispatch(
```

Comment:
Hey @kqualters-elastic Was this the dispatch function to be removed from this hook to fix the data view analyzer problem in the interim?